.cirrus/Dockerfile.ubuntu16.04

```diff
@@ -6,7 +6,7 @@ RUN set -e -x ;\
     apt-get -y update ;\
     apt-get -y upgrade ;\
     apt-get -y install \
-        build-essential cmake clang python3-dev libboost-all-dev qt5-default git
+        build-essential autoconf cmake clang bison wget flex gperf libreadline-dev gawk tcl-dev libffi-dev graphviz xdot python3-dev libboost-all-dev qt5-default git
```

Comment:
While you're at it, please merge this with the packages from the below apt install step (libftdi-dev, pkg-config).